src/Diagnostics.DataProviders/DataProviderConfigurations/SupportObserverDataProviderConfiguration.cs

```diff
@@ -35,6 +35,12 @@ public SupportObserverDataProviderConfiguration()
         [ConfigurationName("IsMockConfigured", DefaultValue = false)]
         public bool IsMockConfigured { get; set; }
 
+        /// <summary>
+        /// Is the local host version of observer enabled.
+        /// </summary>
+        [ConfigurationName("ObserverLocalHostEnabled", DefaultValue = false)]
+        public bool ObserverLocalHostEnabled { get; set; }
```

Comment:
does this require a change in appsettings.json to add the new config?

Reply:
Not really since the default will be false and for us that is okay but it wouldn't hurt to have it there in the appsettings though

Reply:
I asked because sometimes when a property is missing in appsettings.json and we run runtimehost locally, we get a NullReferenceException for the missing config property.

Reply:
Yeah that's good thinking. Because this is bool and is a value type, the value will always be false. And that's what I want it to be for now. I made some changes to the appsettings to include this config. You will see it in the next commit